starter-code/views/celebrities/index.hbs

```diff
@@ -0,0 +1,9 @@
+{{#each celebrity}}
+<a href="/celebrities/{{this._id}}">
+<h2>{{this.name}}</h2></a>
+<form method="POST" action="/celebrities/{{this._id}}/delete">
+<button type="submit" class="btn btn-primary">Delete</button>
+</form>
+
+{{/each}}
+<a href="/celebrities/new">Create new Celebrity</a>
```

Comment:
there is absolutely no indentation in this file. This is very important for code readability.